cg/services/orders/storing/service_registry.py

```diff
@@ -74,6 +74,10 @@ def get_storing_service(self, order_type: OrderType) -> StoreOrderService:
         OrderLimsService,
         StoreCaseOrderService,
     ),
+    OrderType.NALLO: (
```

Comment:
I think we also need to update `cg/services/orders/constants.py:ORDER_TYPE_WORKFLOW_MAP`

Reply:
fixed